Strato/Plist2HTML.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+enum Plist2HTML {
+
```

Comment:
Is there a reason why this is not in the HiDrive-iOS-Common-Framework?

Reply:
I also thought this didn't belong in this submodule. I think we really should have some kind of framework which is shared app code between the iOS and mac app, as the Business framework and lower wouldn't use `Plist2HTML`.